_posts/2019-01-10-Flutter.md

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+
+class JWorksWidget extends StatelessWidget { // I'm a widget
+  @override
+  Widget build(BuildContext context) {
+    return Card( // I'm a widget too
+      elevation: 4.0,
+      child: Padding( // Yep, another widget
+        padding: const EdgeInsets.all(16.0),
+        child: Image.network( // We're all widgets!
+          'https://ordina-jworks.github.io/img/jworks/jworks-400x400.png',
+        ),
+      ),
+    );
+  }
+}
+```
+
+## One hybrid framework to rule them all
+
+On the 4th of December 2018 the first [Flutter Live](https://developers.google.com/events/flutter-live/){:target="_blank" rel="noopener noreferrer"} event was hosted.
+A lot of exciting announcements were made, which definitely shows that the Flutter team is determined to keep improving.
+[The first stable version was released](https://developers.googleblog.com/2018/12/flutter-10-googles-portable-ui-toolkit.html){:target="_blank" rel="noopener noreferrer"} almost two years after the first Alpha version was released.
+
+<img class="image fit" alt="Hummingbird" src="/img/2018-12-17-Flutter/hummingbird.png">
+
+Another huge announcement was revealed about the future plans of Flutter.
+The first step is [Hummingbird](https://medium.com/flutter-io/hummingbird-building-flutter-for-the-web-e687c2a023a8){:target="_blank" rel="noopener noreferrer"}, or Flutter for the web.
+Since Dart can also compile to JavaScript, this was a logical step to take.
+Being able to run on the web, you can also create a Progressive Web App with Flutter, so your mobile app which runs in the browsers becomes a web app.
+Appception right there.
+
+Google is also working to bring Flutter to desktop with Flutter Desktop Embedding.
+To prove this concept, the presentation of Flutter Live was running in a Flutter app on a laptop.
+This means that Flutter won’t be a mobile SDK solely, but it might become the way to go SDK for hybrid apps across mobile, web and desktop.
+Maybe later on, any device that can render pixels will be able to run Flutter apps.
+
+Furthermore, the Flutter team is working to provide integration between Flutter and your existing native apps.
+Not everyone can start from scratch, so having a way to gradually move your app to Flutter is a very welcome addition if you’re planning to do so.
+This project, which contains all the APIs and tooling, is named Add2App and is currently in a preview state.
+With Add2App, you can launch a view containing your new Flutter app from your existing native app.
+
+You can also work in the opposite way with the introduction of Platform Views.
+These views allow you to add native content inside your Flutter apps.
+Platform Views unlocks Flutter to render Google Maps and WebViews inside the Flutter app.
+
+## Conclusion
+
+Flutter keeps getting better and the community keeps on growing.
+More and more companies start to embrace Flutter and developers are excited and positive when using Flutter in their apps.
```

Comment:
Do you have examples?

Reply:
Companies: https://flutter.io/showcase
Community: https://itsallwidgets.com/

Reply:
I'll link them in the blog, thanks!